src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -904,6 +904,8 @@ typedef struct {
 typedef struct {
     RTC_RTP_TRANSCEIVER_DIRECTION direction; //!< Transceiver direction
     RtcRtpReceiver receiver;                 //!< RtcRtpReceiver that has track specific information
+    BOOL firstFrame;
```

Comment:
Do we really need these to be a part of our SDK? Why can't the users just store these values themselves?

Reply:
Because I am calculating it within writeFrame() call. And the packetization happens inside writeFrame. Though, currently, it is one packet per frame usually

Reply:
I would calculate this outside of the SDK, I would add the same logic that you have here but after calling `writeFrame` in the video/audio threads.

Reply:
But what about the case when we have multiple RtpPackets per frame?

Reply:
I think that's okay. Since we're measuring "first frame" delay, fully processing all of the RtpPackets in a frame should be the one that's measured.

Reply:
Hmm. I had measured first RTP packet based on a request. However, since one frame corresponds to one packet mostly, I am measuring first frame delay in application. Since writeFrame could get called before SRTP is set up, I added a new status code to indicate that instead of just returning STATUS_SUCCESS so that the latency is measured appropriately.